tensorflow/python/autograph/pyct/static_analysis/liveness.py

```diff
@@ -65,10 +65,6 @@ def visit_node(self, node):
       reaching_functions = anno.getanno(
           node.ast_node, anno.Static.DEFINED_FNS_IN)
       for fn_ast_node in reaching_functions:
-        if isinstance(fn_ast_node, gast.Lambda):
```

Comment:
Do you think we could inject a deprecation warning here?

Reply:
Sure, though perhaps we should limit it to lambdas which appear inside control flow - the rest are unaffected.

Reply:
Is there quick way to make this extra check? I moved this in `self.lambda_check`